test/driver-loader/runner/test.sh

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2020 The Falco Authors.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+set -euo pipefail
+
+FALCO="falco -M 1"
+FALCO_DRIVER_LOADER=falco-driver-loader
+DRIVER_NAME=falco
+KERNEL_RELEASE=$(uname -r)
+KERNEL_VERSION=$(uname -v | sed 's/#\([[:digit:]]\+\).*/\1/')
+
+
+function get_target_id() {
```

Comment:
Since you have `falco-driver-loader` in the environment where you execute this, you should source it and run its `get_target_id` to actually test it.

Reply:
Yep, I was already thinking about this. I will fix it.

Reply:
Done by https://github.com/falcosecurity/falco/pull/1207/commits/9707568c78a5a28259c7eafab15ef936095da3e6